docs/recipes/quickly-switch between-multiple-foam-repos.md

```diff
@@ -0,0 +1,32 @@
+# Quickly switch between multiple Foam repos
+
+this #recipe is allow to Quickly switch between multiple Foam repos. and sorry for Setting/Content's Hangul the Korean Character.
+
+## Installation
+
+**This extension is not included in the template**
+
+This Method need [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) extension, so please install this extension.
+
+## Quickly switch between multiple Foam repos - Git Folder
+First, Need arrange Quickly switching Multiful Foam repos in one folder
+
+<img width="1322" alt="스크린샷 2021-06-02 오전 10 05 29" src="https://user-images.githubusercontent.com/1904967/120408485-1af37c80-c38a-11eb-8454-713a63ba5c7b.png">
+Second, In VSCode's Extensnion setting, select arrange folder, and extension setting, assign to this folder.
+```Project Manager › Git: Base Folders```
+```Indicates the base folders to search for Git projects```
+
+<img width="1322" alt="스크린샷 2021-06-02 오전 10 06 27" src="https://user-images.githubusercontent.com/1904967/120408569-41b1b300-c38a-11eb-8bc5-9c983600c0d8.png">
+or you can editing project manager's favorites. click to 'Edit Project' and editing ```Project.json``` like this
```

Comment:
Also you backticks are wrong here. This doesn't render correctly at all.

You should use just 1 backtick for `code inside the text` and not 3 of them ````` . 

And when doing a code block, you can use 3 backticks, but they have to be on their own lines:

```

'''
code stuff
comes here
(And imagine that these 3 ' -marks are backticks for you. They are on their own lines with empty lines surrounding them, you see?
'''

```